### DISCUSSION TITLE
2023 edition, Can't interact to anything in a repository as a user

### DISCUSSION THREAD

**Discussion by earllly** [community/community#45301] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

"An owner of this repository has limited the ability to comment from new users. "

Well then how the new user can report a bug or suggest an issue???

https://github.com/DNSCrypt/dnscrypt-proxy/issues/985

@jedisct1

Currently it seems your software is querying 1 server for an answer. If the user selected 'random' or 'ph' as lb strategy, you could query **2** servers at the same time & use the fastest response as final answer.

This will improve the stability of DNS answer & speed. Please consider this.

Currently:

- When the user set 3 servers(A/B/C) and 'random' is selected & B is pretty much unstable --> if dnscrypt select B the final answer will take 2 seconds (B fail -> query C or A)

**Comment by airtower-luna** [comment, marked as answer]
> Well then how the new user can report a bug or suggest an issue???

Wait 24 hours, then you shouldn't count as a new user any more. See the [documentation on interaction limits](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository#about-temporary-interaction-limits):

> Limit to existing users: Limits activity for users with accounts that are less than 24 hours old who do not have prior contributions and are not collaborators.

The point of the setting is to keep out spammers, trolls, harassers etc. who will often try to create a lot of new accounts (because their old ones got blocked).

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @earllly! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
